source/includes/_simulator-reference.html.md

```diff
@@ -54,16 +54,16 @@ Note that config in __main__ is the brain configuration and remains the same thr
 ```
 $ python <simulator_file>.py --help
 usage: <simulator_file>.py [-h] [--accesskey ACCESSKEY] [--username USERNAME]
-                      [--url URL] [--proxy PROXY] [--brain BRAIN]
-                      [--predict [PREDICT]] [--verbose VERBOSE]
-                      [--performance PERFORMANCE]
+                 [--url URL] [--proxy PROXY] [--brain BRAIN]
+                 [--predict [PREDICT]] [--verbose] [--performance]
+                 [--log LOG [LOG ...]] [--record RECORD]
 
 optional arguments:
   -h, --help            show this help message and exit
   --accesskey ACCESSKEY
                         The access key to use when connecting to the BRAIN
                         server. If specified, it will be used instead of any
-                        access key information stored in a bonsai config file.
+                        access keyinformation stored in a bonsai config file.
```

Comment:
lost some whitespace

Reply:
Yeah this unfortunately is in the CLI but I have been too lazy to put in a Diff so I was just changing it in the docs. :P